libraries/provision/ansible/playbooks/tasks/remove-sync-gateway.yml

```diff
@@ -1,18 +1,25 @@
 # Stop sync_gateway service
 - name: SYNC GATEWAY | Stop sync_gateway service
   service: name=sync_gateway state=stopped
-  when: ansible_distribution == "CentOS" and ansible_distribution_major_version == "7"
+  when: (ansible_distribution == "CentOS" and ansible_distribution_major_version == "7") or ansible_distribution == "Ubuntu"
   ignore_errors: yes
 
 - name: SYNC GATEWAY | Stop sync_gateway for CentOS 6
```

Comment:
can you replace centos to generic name as codition is added for other platforms? Make change on other files which has conditions for other platforms but name of the task mentioned as centos